File: ACM/2018新生选拔赛/I was angry! 's easy game/untitled.cpp
#include <iostream>
#include <cstring>
#include <vector>
#include <algorithm>
using namespace std;

int main()
{
    string m, n;
    while (cin >> m >> n)
    {
        vector<int> seq_sum_list;
        // char *m_char = &m[0]; you can convert string to char* by this way.
        for (int i = 0; i < m.length(); i++)
        {
            for (int j = 0; j < n.length(); j++)
            {
                if (m[i] == n[j])
                    seq_sum_list.push_back(i + j);
            }
        }
        if (seq_sum_list.size())
        {
            cout << *min_element(seq_sum_list.begin(), seq_sum_list.end()) << endl;
        }
        else
        {
            cout << "I was angry!" << endl;
        }
    }
    return 0;
}

File: ACM/2018新生选拔赛/I was angry! 's easy game/untitled.py
def culti_num(str):
    seq_sum_list = []
    a, b = str.split(' ')
    l = len(a)
    for i in range(l):
        for j in range(l):
            if a[i] == b[j]:
                seq_sum_list.append(i + j)
    if len(seq_sum_list):
        return min(seq_sum_list)
    else:
        return "I was angry!"

while True:
    try:
        input_str = input()
    except EOFError:
        break
    print(culti_num(input_str))